web/src/apps/selector.ts

```diff
@@ -0,0 +1,179 @@
+import { apply, css, tw } from '@twind/core'
+import { LitElement, html } from 'lit'
+import { customElement, property, state } from 'lit/decorators.js'
+import { Ref, createRef, ref } from 'lit/directives/ref.js'
+
+import { withTwind } from '../twind'
+import type { DocumentView } from '../types'
+/**
+ * This class extends LitElement to include a tw instance.
+ */
+class TWLitElement extends LitElement {
+  protected tw: typeof tw
+}
+
+/**
+ * UI selector
+ *
+ * A selector that updates some display portion of the UI
+ */
+@customElement('stencila-ui-selector')
+@withTwind()
+export class UISelector extends TWLitElement {
+  /**
+   * Ref to allow us to close the details element when needed.
+   */
+  detailsRef: Ref<HTMLDetailsElement> = createRef()
+  /**
```

Comment:
A matter of personal taste but I find the lack of blank lines between properties a little jarring (maybe just because I'm not used to it!)

Reply:
nah - this ain't right. I think this has happened post eslint update. I hate it as well. Will fix!